### DISCUSSION TITLE
Is there any way to change the post slug by adding the tag?

### DISCUSSION THREAD

**Discussion by artxewebponferrada** [GetPublii/Publii#1413] [Help]
I would like to know if there is any way (even modifying the theme files) to use a tag to cluster some pages. 

So that if I have a tag called blog and I include a post it would come out:
domain.com/blog/postname/

**Comment by example-user30** [comment]
URLs are built by publii in the render process as near as I can tell.  There does not seem to be a way to do that.  Perhaps the Devs could add more to this answer.  Sorry

**Reply by artxewebponferrada** [reply]
A functionality similar to the one mentioned above would be ideal. It would allow content grouping, multilingual implementation, etc. Thank you for your response

**Comment by candidexmedia** [comment]
Hi @artxewebponferrada !

There are quite a few threads in the old form on this topic that I have been cataloging over the years (with some replies from the devs), so I will just share them below for you to consult:

https://forum.getpublii.com/topic/folder-structure/

https://forum.getpublii.com/topic/is-there-a-way-to-create-posts-automatically-inside-subfolders/

https://forum.getpublii.com/topic/custom-url-path-for-posts/

Generally speaking, I understand and agree with the reasoning for keeping URLs simple, and not making the main tag part of the URL. If you ever change the name of the tag down the line, you won't be faced with a ton of broken URLs to manage.

# Breadcrumbs for content grouping

In terms of content grouping, I would suggest using breadcrumbs on your page.

![breadcrumbs](https://cv.domagic.site/media/breadcrumbs.jpg)

Here are some posts and forum topics on how to do so:

https://cv.domagic.site/posts/breadcrumbs-for-publii/

https://forum.getpublii.com/topic/breadcrumb-and-multilanguage/

# Multilanguage

A multilingual plugin is on the horizon, [according to Bob](https://forum.getpublii.com/topic/multilanguage-support/#post-11556) on the Publii dev team. The [Publii homepage](https://getpublii.com) makes use of it, and it appears to use folders/subdirectories.

**Reply by artxewebponferrada** [reply]
thank you for your reply! I appear in some of these threads (seb_oi), I would say that we have shared several messages on the forum.

I use modified threads for several things (relevant to seo).

Introducing custom css in a specific post (to avoid unnecessary loading of styles).
Introduce js/html in the head of each post (where I hand insert schemas and some functionalities like some js that have to be loaded in the head to work properly).
The issue of changing the path of the slug in small pages is not necessary but from experience for large pages a clustering is positive for SEO.

Making a multilingual website in theory is not difficult (but the slug issue fails).
-We generate a conditional in the <html lang that can be selected in each language.
-Generate canonicals tags
![publii-multilang](https://github.com/GetPublii/Publii/assets/117656534/fa9b142d-4064-4204-8655-a9c2ef24faf8)

In function of the language add the slug (domain.com/lang/...)
publii-multilang
Breadcrumbs is a great feature.
